index.html

```diff
@@ -16,147 +16,180 @@
-        <div class="card-flex">
-          <div class="description">
-            <h2 class="new">Multi-Post Stories Gain+Glory</h2>
-            <ul class="list p-lang">
-              <li>
-                <button class="btn-btn" type="button">Ruby on rails</button>
+              <li class="intro-contact">
+                <a href="https://twitter.com/brown_admirer"
+                  ><i class="fa-brands fa-twitter"></i
+                ></a>
               </li>
-              <li><button class="btn-btn" type="button">Css</button></li>
-              <li><button class="btn-btn" type="button">Javascript</button></li>
-              <li><button class="btn-btn" type="button">Html</button></li>
             </ul>
-            <button class="btn view" type="button">See Project</button>
           </div>
-        </div>
-        <div class="card-flex">
-          <div class="description">
-            <h2 class="new">Multi-Post Stories Gain+Glory</h2>
-            <ul class="list p-lang">
-              <li>
-                <button class="btn-btn" type="button">Ruby on rails</button>
-              </li>
-              <li><button class="btn-btn" type="button">Css</button></li>
-              <li><button class="btn-btn" type="button">Javascript</button></li>
-              <li><button class="btn-btn" type="button">Html</button></li>
-            </ul>
-            <button class="btn view" type="button">See Project</button>
+          <div class="headline-button">
+            <img src="images/Action.svg" />
           </div>
         </div>
-        <div class="card-flex">
-          <div class="description">
-            <h2 class="new">Multi-Post Stories Gain+Glory</h2>
-            <ul class="list p-lang">
-              <li>
-                <button class="btn-btn" type="button">Ruby on rails</button>
-              </li>
-              <li><button class="btn-btn" type="button">Css</button></li>
-              <li><button class="btn-btn" type="button">Javascript</button></li>
-              <li><button class="btn-btn" type="button">Html</button></li>
-            </ul>
-            <button class="btn view" type="button">See Project</button>
+      </div>
+    </section>
+
+    <!-- recent works section -->
+    <section class="call">
+      <div id="recent-works">
+        <div class="section-tag">
+          <h1 class="section-title">My Recent Works</h1>
+          <a><img src="images/Indicator.svg" alt="Indicator" /></a>
+        </div>
+
+        <div class="high-container">
+          <div class="grid-container">
+            <div class="card-flex">
+              <div class="description">
+                <h2 class="new">Multi-Post Stories Gain+Glory</h2>
+                <ul class="list p-lang">
+                  <li>
+                    <button class="btn-btn" type="button">Ruby on rails</button>
+                  </li>
+                  <li><button class="btn-btn" type="button">Css</button></li>
+                  <li>
```

Comment:
You have done great work implementing the button interactions. The languages listed are not button and so are not meant to have the interations. You have added a hover.

[Optional] I recommend that you remove the `button` tag within the `li` tags for each of them and allow them be directly in the `li` tags. This is more semantic.

Your design:
![image](https://user-images.githubusercontent.com/20554583/200075248-84a057ad-c0de-45df-b684-761fe7db943d.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/200075306-50adb3a0-5fd1-46c1-bc30-a252b5b9bd7e.png)